docs/auth-openstack.md

```diff
@@ -0,0 +1,59 @@
+# Authn/Authz
+Users of the Inventory can provide OpenStack service specific credentials,
+which are then used against the specified OpenStack Keystone service.
+Multiple credentials can be provided per service, with the unique property
+being project_id, which is expected to be unique globally. With this setup,
+users can collect resources from multiple regions/clusters, as long as the
+credentials are configured properly.
+
+Make sure the credentials you provide have the roles needed to access the 
+respective resources. If there are resources with different policies in the 
+same service type that need to be collected, you can list different credentials
+for both. This might cause an error in inventory logs, but won't stop
+the collection process.
+
+A full example can be found at `./examples/config.yaml`
+
+Example credentials section:
+```
+credentials:
+  local:
+    authentication: password
+    password:
+      username: "I748922"
+      password_file: "./local_pass"
+  named_credentials_myproject
+    authentication: app_credentials
+    app_credentials:
+      app_credentials_id: <id>
+      app_credentials_name: <credentials_name>
+      app_credentials_secret_file: <secret>
+```
+
+The two supported credential types are `username` and `app_credentials`.
+
+The service section must also be configured.
+Example services section populated for the network service:
+
+```
+services:
+  network:
+    - use_credentials: named_credentials_myproject #defined above
+      domain: <domain>
+      region: <region>
+      project: <project name>
+      project_id: <project id of the project above>
+      auth_endpoint: https://<keystone-url>
+  compute:
```

Comment:
The paragraph above states that the example is for the `network` service, so perhaps remove the `compute`, `load_balancer` and `identity` sections from the config in order to avoid confusion, or fill them in.